framework/include/cppmicroservices/detail/BundleAbstractTracked.tpp

```diff
@@ -31,23 +31,25 @@ namespace detail {
 
 template<class S, class TTT, class R>
 BundleAbstractTracked<S,TTT,R>::BundleAbstractTracked(BundleContext bc)
-  : closed(false), trackingCount(0), bc(bc)
+  : closed(false)
+  , trackingCount(0)
+  , bc(bc)
```

Comment:
This works but I'd recommend changing the parameter name from `bc` to something else as the private member is also called `bc`. This should help with code readability.

Reply:
Changed to `context`